uk.co.compucorp.civicrm.tasksassignments/tests/phpunit/CRM/Activity/Service/ActivityServiceTest.php

```diff
@@ -33,40 +33,50 @@ function testFindByComponentLimitingResultsAmount() {
   /**
    * @expectedException CiviCRM_API3_Exception
    */
-  function testFindByComponentFetchingByInvalidComponent() {
+  public function testFindByComponentByInvalidComponentThrowsError() {
     $activities = ActivityService::findByComponent('NonExistingComponent');
   }
 
-  function testCheckIfActivityTypeBelongsToComponent() {
-    $activityTypeId = array_keys(civicrm_api3('Document', 'getoptions', [
-      'field' => 'activity_type_id'
-    ])['values'])[0];
-
-    $activityBelongsToDocumentComponent =
-      ActivityService::checkIfActivityTypeBelongsToComponent(
-        $activityTypeId, 'CiviDocument');
-    $activityBelongsToTaskComponent =
-      ActivityService::checkIfActivityTypeBelongsToComponent(
-        $activityTypeId, 'CiviTask');
+  public function testGetTypesIdsForComponent() {
+    $typesIds = ActivityService::getTypesIdsForComponent('CiviDocument');
 
-    $this->assertTrue($activityBelongsToDocumentComponent);
-    $this->assertFalse($activityBelongsToTaskComponent);
+    foreach($typesIds as $typeId) {
+      $this->assertTrue(ActivityService::isDocumentComponent($typeId));
+    }
   }
 
-  function testGetTypesIdsForComponent() {
-    $typesIds = ActivityService::getTypesIdsForComponent('CiviDocument');
+  public function testIsTaskComponentReturnsTrueForTaskTypeIds() {
+    $documentTypesIds = ActivityService::getTypesIdsForComponent('CiviTask');
```

Comment:
Variable name should probably be `$activityTypesIds`

Reply:
Done.